airbyte-integrations/bases/base-normalization/unit_tests/test_table_name_registry.py

```diff
@@ -161,28 +165,37 @@ def test_get_simple_table_name(json_path: List[str], expected_postgres: str, exp
     actual_bigquery_name = bigquery_registry.get_simple_table_name(json_path)
     assert actual_bigquery_name == expected_bigquery
 
+    oracle_registry = TableNameRegistry(DestinationType.ORACLE)
+    actual_oracle_name = oracle_registry.get_simple_table_name(json_path)
+    assert actual_oracle_name == expected_oracle
+    assert len(actual_oracle_name) <= ORACLE_MAX_TABLE_LENGTH
+
 
 @pytest.mark.parametrize(
-    "json_path, expected_postgres, expected_bigquery",
+    "json_path, expected_postgres, expected_bigquery, expected_oracle",
     [
         (
             ["parent", "child"],
             "parent_30c_child",
-            "parent_30c_child",
+            "parent__30c_child",
+            "PARENT_30C_CHILD",
```

Comment:
This is the part I change the function `json_path_hash = name_transformer.normalize_table_name(hash_json_path([schema] + json_path), False, False)`
but changing it also change BigQuery table name.